### ISSUE TITLE
Paged tables too big to fit on revealjs slides

### ISSUE THREAD

**Issue by davidbudzynski** [quarto-dev/quarto-cli#4328]
### Bug description

When trying to insert paged table: the output is too big to fit on the screen:

[Source to reproduce](https://pastebin.com/A2zTMfqY)

Result with 100% page zoom:
![image](https://user-images.githubusercontent.com/56514985/218330609-e41942e7-be39-4835-9b5d-e4daecc99b7a.png)

Result with 50% page zoom:
![image](https://user-images.githubusercontent.com/56514985/218330662-2dd2f31e-0428-4ce6-9477-2b8157268c41.png)

Session info:

```
R version 4.2.2 (2022-10-31)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Arch Linux

Matrix products: default
BLAS:   /usr/lib/libopenblas.so.0.3
LAPACK: /usr/lib/liblapack.so.3.11.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_GB.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_GB.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_GB.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_GB.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] quarto_1.2   httpgd_1.3.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.9        ps_1.7.2          digest_0.6.31     later_1.3.0       parallelly_1.33.0
 [6] jsonlite_1.8.4    evaluate_0.19     rlang_1.0.6       cli_3.5.0         rstudioapi_0.14  
[11] rmarkdown_2.19    tools_4.2.2       yaml_2.3.6        fastmap_1.1.0     xfun_0.36        
[16] parallel_4.2.2    compiler_4.2.2    processx_3.8.0    systemfonts_1.0.4 htmltools_0.5.4  
[21] knitr_1.41       
```

Rstudio version:

```
RStudio 2023.03.0-daily+265 "Cherry Blossom" Daily (df867f7d2dc6f7a6ddad4b6cdea992fa41c182a1, 2023-01-20) for Ubuntu Jammy
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) rstudio/2023.03.0-daily+265 Chrome/108.0.5359.179 Electron/22.0.3 Safari/537.36
```

### Checklist

- [X] Please include a minimal, fully reproducible example in a single .qmd file? Please provide the whole file rather than the snippet you believe is causing the issue.
- [X] Please [format your issue](https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us) so it is easier for us to read the bug report.
- [X] Please document the RStudio IDE version you're running (if applicable), by providing the value displayed in the "About RStudio" main menu dialog?
- [X] Please document the operating system you're running. If on Linux, please provide the specific distribution.

**Comment by cscheid** [comment]
Is this a duplicate of https://github.com/quarto-dev/quarto-cli/issues/3380, in which case it should be fixed on the 1.3 prereleases?

**Comment by davidbudzynski** [comment]
I'll check 1.3 and reopen if needed, thanks!
